content/sensu-go/6.2/operations/deploy-sensu/use-federation.md

```diff
@@ -312,24 +289,24 @@ spec:
   }
 }
 {{< /code >}}
-
 {{< /language-toggle >}}
 
-To configure this etcd replicator on your `gateway` cluster, use `sensuctl config view` to verify that `sensuctl` is configured to talk to a `gateway` cluster API. Reconfigure `sensuctl` if needed.
+2. Run `sensuctl config view` and verify that `sensuctl` is configured to talk to a `gateway` cluster API.
+Reconfigure `sensuctl` if needed.
 
-Write these EtcdReplicator definitions written to disk and use `sensuctl create -f` to apply them to the `gateway` cluster.
+3. Write the `AlphaClusterRoleBindings` and `BetaClusterRoleBindings` EtcdReplicator definitions to disk.
 
-For a consistent experience, repeat the `ClusterRoleBinding` example in this guide for `Role`, `RoleBinding` and `ClusterRole` resource types.
-The [etcd replicators reference][2] includes [examples][9] you can follow for `Role`, `RoleBinding`, `ClusterRole`, and `ClusterRoleBinding` resources.
+4. Use `sensuctl create -f` to apply the `AlphaClusterRoleBindings` and `BetaClusterRoleBindings` EtcdReplicator definitions to the `gateway` cluster.
 
-To verify that the EtcdReplicator resource is working as expected, reconfigure `sensuctl` to communicate with the `alpha` and then `beta` clusters and run the following command for each:
+5. For a consistent experience, create an EtcdReplicator resource for `Role`, `RoleBinding` and `ClusterRole` resource types.
+The [etcd replicators reference][2] includes [examples][9] you can follow for `Role`, `RoleBinding`, and `ClusterRole` resources.
 
+6. Verify that the EtcdReplicator resource is working as expected: reconfigure `sensuctl` to communicate with the `alpha` and then `beta` clusters and run the following command for each:
```

Comment:
What does it mean to reconfigure sensuctl to communicate with the alpha and beta clusters? Is it just running sensuctl configure again after creating the etcd replicators?

Reply:
@hillaryfraley in this case the instructions seem to mean that one needs to reconfigure `sensuctl` using the connection details for the `alpha` and `beta` clusters so you can then run that command on each of them directly.